indico/modules/events/registration/blueprint.py

```diff
@@ -84,6 +86,8 @@
 # Management
 _bp.add_url_rule('/manage/registration/', 'manage_regform_list', RHManageRegistrationForms)
 _bp.add_url_rule('/manage/registration/create', 'create_regform', RHRegistrationFormCreate, methods=('GET', 'POST'))
+_bp.add_url_rule('/manage/registration/display',
+                 'manage_regforms_display', RHManageRegistrationFormsDisplay, methods=('GET', 'POST'))
```

Comment:
Not a strong opinion here, but I would break the line after the RH class if possible.